UniConnectApp/app/src/main/java/com/example/uniconnect/dao/IUniversityService.kt

```diff
@@ -1,4 +1,4 @@
-package com.example.uniconnect.service
+package com.example.uniconnect.dao
```

Comment:
Since this is named IUniversityService, it almost definitely belongs in a service package, not a DAO package.